Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -361,6 +385,12 @@ public void Scan()
 
                     foreach (var fill in fills)
                     {
+                        // check if the fill should be emitted
+                        if (!timeInForceHandler.IsFillValid(security, order, fill))
+                        {
+                            break;
```

Comment:
OK for now, but this will change, we can't assume the implementation of `IsFillValid` negates all fills in the event there are multiple fills -- when we wire up order event projections this will change, so it's ok for now.